policyengine_us/variables/household/expense/housing/rent.py

```diff
@@ -7,3 +7,15 @@ class rent(Variable):
     label = "Rent"
     unit = USD
     definition_period = YEAR
+
+    def formula(person, period, parameters):
+        pre_subsidy_rent = person("pre_subsidy_rent", period)
+        total_pre_subsidy_rent = person.spm_unit.sum(pre_subsidy_rent)
+        housing_subsidy = person.spm_unit("housing_assistance", period)
+        rent_fraction = np.zeros_like(total_pre_subsidy_rent)
```

Comment:
Suggest allocating `total_rent` from here
```suggestion
        total_rent = total_pre_subsidy_rent - housing_subsidy
        rent_fraction = np.zeros_like(total_pre_subsidy_rent)
```

Reply:
added but I left the max statement to avoid the case that users return a negative rent value if the housing assistance is entered manually